tools/packaging/kernel/build-kernel.sh

```diff
@@ -399,6 +419,9 @@ build_kernel() {
 	arch_target=$(arch_to_kernel "${arch_target}")
 	pushd "${kernel_path}" >>/dev/null
 	make -j $(nproc) ARCH="${arch_target}"
+	if [ "${conf_guest}" == "sev" ]; then
+		make -j $(nproc) INSTALL_MOD_STRIP=1 INSTALL_MOD_PATH=${kernel_path} modules_install
```

Comment:
Nit: This is a potential DoS scenarios when run on a dev box. I'd suggest something like:

```suggestion
		make -j $(nproc --ignore=1) INSTALL_MOD_STRIP=1 INSTALL_MOD_PATH=${kernel_path} modules_install
```

... but that would require an extra commit to tweak `build_kernel()` which has the same issue.

Reply:
Sure, I can tweak this PR to exclude one processor like you suggested and also look at opening an issue for the default make to avoid the same issue.